cmd/README.md

```diff
@@ -0,0 +1,655 @@
+# Precompile Generation Tutorial
+A stateful precompile allows us to add more functionality and customization to the EVM. A stateful precompile builds on a precompile in that it adds state access. This means our precompile can manage state as well as interacting with EVM state. 
+
+A stateful precompile follows this interface. 
```

Comment:
This one follows the same format as well and doesn't provide much information. What is the purpose of this interface?

Reply:
Bumping this comment